plugin-gradle/compat/java/src/main/kotlin/org/jsonschema2dataclass/internal/compat/java/JavaPluginRegistration.kt

```diff
@@ -41,12 +41,10 @@ class JavaPluginRegistration : GradlePluginRegistration {
             }
 
             dependsOn.execute("compileKotlin")
-            dependsOn.execute("compileJava")
```

Comment:
No need to declare this task dependency explicitly anymore because it's now derived from source set configuration as it should be. I believe that the dependency from `compileKotlin` is also unnecessary but I will leave adding a test for that and verifying it to you, @eirnym. I suspect that this is also the case for the lombok task as long as it declares the source set as it's input.

Reply:
Done